lib/internal/process/execution.js

```diff
@@ -42,14 +42,10 @@ function evalModule(source, print) {
   if (print) {
     throw new ERR_EVAL_ESM_CANNOT_PRINT();
   }
-  const { log } = require('internal/console/global');
   const { loadESM } = require('internal/process/esm_loader');
   const { handleMainPromise } = require('internal/modules/run_main');
   return handleMainPromise(loadESM(async (loader) => {
-    const { result } = await loader.eval(source);
-    if (print) {
-      log(result);
-    }
+    await loader.eval(source);
```

Comment:
```suggestion
    return loader.eval(source);
```
And you could remove the `async` above.

Reply:
It’s not equivalent if `loader.eval()` fulfills with something other than `undefined`. Is there a reason to prefer not having `async` in the function declaration?

Reply:
> It’s not equivalent if loader.eval() fulfills with something other than undefined.

I had looked at the code of `loadESM`, the result of the callback is not used anyway.

> Is there a reason to prefer not having async in the function declaration?

It's just useless in this case.